apps/src/code-studio/pd/form_components/FormController.jsx

```diff
@@ -338,6 +338,8 @@ export default class FormController extends React.Component {
         pageData[field] = value;
       }
     });
+
+    pageData = {...pageData, ...currentPage.processPageData(pageData)};
```

Comment:
Can this just be done with `Object.assign(pageData, currentPage.processPageData(pageData)` ?

Reply:
yeah that should work. 👍